src/utils.ts

```diff
@@ -36,7 +36,20 @@ export function nMsAgo(n: number): number {
 export function wait(n: number): Promise<void> {
 	return new Promise((resolve) => setTimeout(resolve, n));
 }
-
+export function humanReadableSize(
+	bytes: number,
+	dm: number,
+	ibi: boolean = false
+) {
+	const k = ibi ? 1024 : 1000;
+	const sizes = ibi
+		? ["B", "kiB", "MiB", "GiB", "TiB"]
+		: ["B", "kB", "MB", "GB", "TB"];
+
+	const i = Math.floor(Math.log(bytes) / Math.log(k));
+
+	return `${parseFloat((bytes / Math.pow(k, i)).toFixed(dm))} ${sizes[i]}`;
```

Comment:
I don't think it's really worth it to make this customizable—it just makes it more likely that we'll introduce inconsistencies down the road. this function only really needs a single argument - `dm` can probably be hardcoded to 2